README.md

```diff
@@ -40,10 +49,10 @@ Stamps have a method called `.compose()`:
 stamp.compose(...args?: stamp|descriptor[]): stamp
 ```
 
-When called the `.compose()` method creates new composable using the current composable as a base, composed with a list of *composables* or *descriptors* passed as arguments:
+When called the `.compose()` method creates new stamp using the current stamp as a base, composed with a list of *composables* or *descriptors* passed as arguments:
```

Comment:
`or *descriptors*` should be dropped.
Because you just decided that Descriptor is Composable too.
Btw, you have decided that on your own.
I like the idea to declare that Descriptors are Composables too, but next time lets discuss terminology changes at least in the gitter chat.